notebooks/timeseries.ipynb

```diff
@@ -0,0 +1,568 @@
knZsibMm/c/evRoxuTJv9K3rxtjxnTlyy97gWkpFakifTi+EhNTCAj4N2v4xbVrDwCoUsWMTp0a0qVLIzo2q4qlZSVMq1fXcrXq04d9ogpdy5E5dKN9+7eYOTN96Iab26BcQ6ryfK6OZSkupeSA4mWZM+f/h2iYmOhGE+p13yelTTf2uo5Swn+goJwckDvLli3n+eorHzp1aoCn50i9GdKg6j7p0KE+/v5fsnDhPtzdT3L8+E0SfiqLacv4Eq5Qdbp4fMky3PknPKOhfJsLF+4SH5+EkZEBrVvXZcaM3tjaNqJFi9oYGipv+IUu7pPi0NUc9vZtsLGxwsXFk4ED1zNxoh1Tp/Ys8FjS1SxFpZQcUPQsmUM0pkzRrSEar/M+0QbRcC5AakICAIamOtbFV0RKyQE5s6xde4JFi37Dzq4Z69cPxdRUf+5zWpR9Ur68KcuWOfDhhy2YOnUH//VpRFXX/4j7IgkzM+3PdalLx9ezZzE8cavFy98qY/toKQD16lng5NQWW9tGdOxYn/Ll865Tl3KoSylZdDlHgwbVOXBgMl9/7ceqVUe5dOkea9YMyvfaCl3OUhRKyQFFy/L8efosGk2b1mTCBN0aovG67hNtEQ3nAkTfuAFo5j722qSUHJCeRQY2n4hg5cqjfPLJ26xe7ax3VzUXZ59069aEEyem027OWCJXWtJ5+7d8+WUv7O3baLXXVBeOr7i4JDZuPI2b23Fi4i0o3yOauZOc6dKlEVZW1VRahy7k0BSlZNH1HGZmJqxYkT7rRubQjdWr8x66oetZVKWUHFC0LHPn7iYyMpZfftGdIRqZXtd9oi3K+4xSUDQZ2LzpDCtXHsXJqR1r1gzSu0azOipXNsNydSi1/YKpUaMSU6Zsx85uOSdO3CJ94prXS2pqGt7eF+nU6VuWLj1Ip04NsT5xi5qb/mXQoI4qN5oFQR329m04dGgyVauWY+DA9Xz33SFSU8VtzJXiyJHr+PoG4ur6gU4N0RC0QzScBb2RmprG2rUn2LsviJEjO7N8uYMix6eqwqx9LPv3T2L9+qEkJCQzaJAHjo7uXL16X9ullQpZlvH3v8EHH3zPtGk7qFWrMrt3u7J58whMGiRquzzhNdSwYQ0OHpyMvX0bVq06iqOjO+Hh0douS1BT+iwaO2natCYTJ4obnQii4SzoiZSUVCZO9ObIkesZd9brq9dz6iY9f06AqysJT54Uex2SJPHxxzacOjWDRYv6cfPmIz78cAXjxm0jLOyZBqvVLUFBYdjbr2XIkI0kJ6eyYcMw9u2bSLt2Yro+QbvMzMqwatUAVq4cwOXLodjZLefMmb+1XZaghnnz9hAZGcvKlQN0boiGoB362/IQXhuJiSm4uGzBzy+QIYM7MnhQRzLuUqm3wnx9SQwPJ3j1arXXZWJixIgRnfn996+ZONGOw4ev0bnzEubPTz/hlzRN/BKgipCQp4wZs5XevVfy99//8e23n3Hy5Aw++qiV3h8PgrI4OrbNGroxYMB6vv/+EGlpr99QKn139Oh1fHwCcHX9gBYtdH+aNKF0iF+fClCuTh1tl6AR+pwjLi6JkSM3c/r03yxa1A/nT5pouyS1JUREEHHqFAD3fX1p6OqKqYWF2uutUMGUGTN6M2RIR3744QgbN55h+/Y/GD/+A0aO7EzZsiUzA8fjI0eyfglouWCBxtf/7FkMP/54jC1bzmNkZMikST0YN65bvrNjFJc+v09epZQs+pyjUSNLDh6czKxZvqxceZSr5y1ZsqQ/VbVdmJr0eZ+8qqAsz5/HMn36Tpo0sdT5IRqvyz7RFaLHuQBlzM0pY26u7TLUpq85Xr5MwNl5PWfPBrNihRMjRnTW2yzZBbu5pU80DMipqRrpdc7O0rIyy5c7cvz4l7RtW49vv91Pp05L2LnzksYvWEqIiODRgQNA+i8Bmux1jotLYvVqfzp2XMzmzWdxdGzLhQuzmD69l8YbzaC/75O8KCWLvudIH7oxkJUrB3D+6lM+cvLk7NlgbZelFn3fJ9kVlGXevD08e6YfQzRel32iK0TDuQApsbGkxJb8R90lTR9zpDwzwsFhLYGBIaxdOxgnp3bpj+thluwSIiK47+ODnJwMgJycrPEGZ6ZGjSzZunUUPj5fUL16RSZN+pWePX/g1KnbGpuBI9jNjbTUVEBzvwSkpqaxffsfdO78LUuWHKBDh/qcPDmD775zyHeO3ExeeHGRi5zmNNZY44WXytvV92MrO6VkUUoOR8e2HPAbjUVFY5yc1rF8+WG9nHXDCy8axlpjFmtY5PeXLsrv+Mo+RKNly13MIQkAACAASURBVDe1UFnRKOV9AvqRRTScC/Di9m1e3L6t7TLUpm85ku+bcP9/Dfj77//YtGkEn3zydtbP9C3Lq4Ld3JDTcv6HWRK9ztl17FifAwcmsW7dEGJjExk4cD1OTuuybjtdXJm/BJCSAqj/S4Asy5w4cQs7u+VMmbKdGjUq4ec3Hk/PkTRoUPjdpLzwwgUXEkmfVSOUUFxwUfk/d30/trJTShal5AConhrF1mV29O//LitWHMHJyZ2IiBfaLktlme8vw9v/Uee2cZHfX7oor+MrKiqOGTN20aSJJZMm6fYQjUxKep/oQxbd/vxBeK2kTzF2k7AZDZDjDdi1fQxt2yprpoRwf/+s3uZMcnIy4f7+UITxwUEE0ZWuqm9YAj4Bkw8lLLaac35lCj17BlOhXyTmMx5j/GaS6uvKEBsbQoJnBFa3008jYQ2TwUDCNKYN5Sysi7SuhKtlebKoFvHnKmBcNwFLj8fEfHSFmdJelddxkYtZjeZMccQxm9k441ykeoTXW5HfXyqoSSKUgUc/lqF6x6pc+CqNd+1mYbkmBLNOMRrdVkn4//dXmazHlPj+mjdvN0+fxrBly+c6P0RD0A5xVAg64cqVUBYu/I2LF+9iXC+Nmt53adtYWY1mALsLFwCIDAwEind3pIEMLPb2JROZKp8/oaLDM56vqc7zDW8Qs78yl
```

Comment:
I think we should make clear above that this is only to show how the interpolation is done and that protected members should not be used by the end-user.

Reply:
Ok how about a note at the top like, "This notebook accesses a number of private methods and attributes of OpenSCM classes. We do this to illustrate how OpenSCM works under the hood. However, the user should not access these private classes as these APIs are likely to change without warning. If you think one of these private classes should become public, please raise an issue at https://github.com/openclimatedata/openscm/issues."

Reply:
Perfect!